src/export/InstanceExporter.ts

```diff
@@ -51,14 +51,27 @@ export class InstanceExporter {
             .join('.');
           rulePathPart += '.';
 
-          this.setFixedValuesForDirectChildren(
+          this.setFixedValuesFromStructureDefinition(
             element,
             rulePathPart,
             instanceDef,
             instanceOfStructureDefinition
           );
         }
+      } catch (e) {
+        logger.error(e.message, rule.sourceInfo);
+      }
+    });
 
+    // All rules will be FixValueRule
+    fshInstanceDef.rules.forEach(rule => {
+      rule = replaceReferences(rule, this.tank, this.fisher);
```

Comment:
It's not a super big deal, but I noticed you make this function call as the first thing you do in both times you iterate over `fshInstanceDef.rules`.  I think you could eliminate this redundant processing by doing this before you iterate over any rules (i.e., around line 30):

```ts
const rules = fshInstanceDef.map(r => replaceReferences(r, this.tank, this.fisher));
```

Then iterating over `rules` instead of `fshInstanceDef.rules` in these two loops.